browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -298,6 +298,8 @@ void RewardsDOMHandler::OnWalletProperties(
     }
 
     result.SetDictionary("wallet", std::move(walletInfo));
+    result.SetDouble("monthlyAmount", wallet_properties->montly_amount);
```

Comment:
per comment from @bbondy, could a test be added here to ensure `monthlyAmount` is set?